client/components/thank-you/theme.ts

```diff
@@ -0,0 +1,37 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
+	colors: { textColorLight: string; borderColorLight: string; studioGrey: string };
+	breakpoints: {
+		desktopUp?: string;
+		tabletUp?: string;
+		bigPhoneUp?: string;
+		smallPhoneUp?: string;
+		tabletDown: string;
+	};
+	weights: {
+		bold: string;
+		normal: string;
+	};
+	fontSize: {
+		small: string;
+	};
+};
```

Comment:
I'd like to suggest we move all type definitions in a new `types.ts` file (here [an example](https://github.com/Automattic/wp-calypso/blob/22cb668cceb98d0a166f3fe23b38ec5e1aa917dc/packages/calypso-products/src/types.ts) from a different package). This includes those two ones plus the three other ones in `index.tsx`. That way we have all the types located in a central place. This would also reduce the length of our files a bit.